addons/website_event/models/event_event.py

```diff
@@ -460,12 +460,9 @@ def _track_subtype(self, init_values):
             return self.env.ref('website_event.mt_event_unpublished', raise_if_not_found=False)
         return super()._track_subtype(init_values)
 
-    def _get_external_description(self):
-        """ Adding the URL of the event into the description """
-        self.ensure_one()
-        event_url = f'<a href="{self.event_register_url}">{self.name}</a>'
-        description = event_url + '\n' + super()._get_external_description()
-        return description
+    def _get_event_url(self):
```

Comment:
Yes it seems we should have a single "url" field people use and modify (based on input or website url).

Reply:
As discussed, it's too finicky to try to tell whether the url needs to be updated or not (we don't want to overwrite user inputs). Switched to a computed field.
+ rebased